ttml.go

```diff
@@ -204,6 +208,51 @@ type TTMLInDuration struct {
 // - hh:mm:ss:fff (fff being frames)
 func (d *TTMLInDuration) UnmarshalText(i []byte) (err error) {
 	var text = string(i)
+	if matches := ttmlRegexpOffsetTime.FindStringSubmatch(text); matches != nil {
+		metric := matches[4]
+		value, err := strconv.Atoi(matches[1])
+
+		d.d = time.Duration(0)
+
+		var (
+			nsBase       int64
+			fraction     int
+			fractionBase float64
+		)
+		if len(matches[3]) > 0 {
```

Comment:
Could you return an error in the `else` statement?

Reply:
this is not an error, if the fraction ( .001) part has 0 length, fraction and fractionBase will simply be 0